src/renderer/components/debug/DebugMock.js

```diff
@@ -348,6 +352,8 @@ const DebugMock = () => {
     notifExtra,
     notifLanguages,
     notifPlatform,
+    notifAppVersionApps,
+    notifLLComonVersionApps,
```

Comment:
Why not `notifLiveCommonVersions` and `notifAppVersions`, in any case case you are missing an **m**

Reply:
Renamed, thanks ✅